data/scripts/combine_projects.py

```diff
@@ -35,16 +36,24 @@ def clean_number(num_str):
 
 def get_census_tract(lat, long):
     """use the census geocoder to find a tract based on lat/long"""
-    try:
-        r = session.get(
-            f"https://geocoding.geo.census.gov/geocoder/geographies/coordinates?benchmark=4&format=json&vintage=4&x={lat}&y={long}")
-
-        json_val = r.json()
-        return json_val["result"]["geographies"]["Census Tracts"][0]["GEOID"]
 
-    except KeyError:
-        print("Failed to find census tract", lat, long)
-        return "00000"
+    max_retries = 5
+    delay_seconds = 2
+    for attempt in range(1, max_retries + 1):
```

Comment:
census API was returning errors, so added some retry code to ensure we get a response